pkg/controller/config/config_controller.go

```diff
@@ -354,11 +372,11 @@ func (r *ReconcileConfig) applyCommunityResources(req reconcile.Request, cfg *op
 	}
 	log.Info("successfully applied all non Red Hat resources")
 
-	err := r.updateStatus(cfg, op.ConfigCondition{Code: op.InstalledStatus, Version: flag.TektonVersion})
+	err = r.updateStatus(cfg, op.ConfigCondition{Code: op.InstalledStatus, Version: flag.TektonVersion})
 	return reconcile.Result{Requeue: true}, err
 }
 
-func transformManifest(cfg *op.Config, m *mf.Manifest, addnTfrms ...mf.Transformer) error {
+func transformManifest(cfg *op.Config, m *mf.Manifest, addnTfrms ...mf.Transformer) (mf.Manifest, error) {
```

Comment:
how about we return `*mf.Manifest, error`?

Reply:
ok .. scratch that .. `m.Transform` returns mf.Manifest

Reply:
yes that is the change happened in dependency